service/service.go

```diff
@@ -10,6 +10,7 @@ import (
 	"golang.org/x/crypto/bcrypt"
 )
 
+//go:generate mockgen -source=service.go -destination=mocks/mock.go
```

Comment:
target filename should have `_test` prefix and include the name of an interface. For example `repo_mock_test.go`.

Reply:
this comment should be to each interface you want to mock, not to struct. 
And you should definitely specify what interface you want to use for generation.